packages/css/src/components/breadcrumb/breadcrumb.scss

```diff
@@ -25,27 +25,22 @@
 .amsterdam-breadcrumb__list {
   break-after: avoid;
   break-inside: avoid;
-  display: flex;
-  flex-wrap: wrap;
-  gap: 0.5rem;
-  list-style: none;
 
   @include reset;
 }
 
 .amsterdam-breadcrumb__item {
-  align-items: center;
-  display: flex;
+  display: inline;
 }
 
 .amsterdam-breadcrumb__item:not(:last-child)::after {
   background-color: var(--amsterdam-breadcrumb-separator-background-color);
-  clip-path: path("M 4.725,16 3.275,14.545 9.815,8 3.275,1.455 4.725,0 l 8,8 z");
+  clip-path: path("M2.725 8 2 7.272 5.27 4 2 .728 2.725 0l4 4-4 4Z");
   content: "";
   display: inline-block;
-  height: 1rem;
-  margin-inline-start: 1rem;
-  width: 1rem;
+  height: 0.5rem;
```

Comment:
Are we sure about this size? It looks really tiny imo

Reply:
It matches the current design. Doesn’t bother me very much, but I can imagine we make the chevrons as large as the font’s x-height. I’ll check with design.

(Although making it respond to font size requires a different approach, the current path is in static pixels.)

Reply:
Chevrons will become as tall as the x-height of the test. I’ll update this PR.

Reply:
Nice!